contrib/platform/src/com/sun/jna/platform/unix/LibCAPI.java

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2015 Goldstein Lyor, All Rights Reserved
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
+package com.sun.jna.platform.unix;
+
+import com.sun.jna.Library;
+
+/**
+ * Note: we are using this &quot;intermediate&quot; API in order to allow
+ * Linux-like O/S-es to implement the same API, but maybe using a different
+ * library name
+ * @author Lyor Goldstein
+ */
+public interface LibCAPI extends Reboot, Resource, Library {
```

Comment:
Should probably avoid implementing `Library` except on the final interface class that's going to be defining an `INSTANCE` variable.